docs/MO_DG/prepare_model/convert_model/Cutting_Model.md

```diff
@@ -1,49 +1,49 @@
 
-Model cutting is illustrated with Inception V1. This model is in `models/research/slim` repository. [This section](Converting_Model.md) describes pre-work to prepare the model for the Model Optimizer to be ready to proceed with this chapter.
+Model cutting is illustrated with Inception V1. This model is in *`models/research/slim`* repository. [This section](Converting_Model.md) describes pre-work to prepare the model for Model Optimizer to be ready to proceed with this chapter.
 
 ## Default Behavior without --input and --output
 
-The input model is converted as a whole if neither `--input` nor `--output` command line options are used. All `Placeholder` operations in a TensorFlow\* graph are automatically identified as entry points. The `Input` layer type is generated for each of them. All nodes that have no consumers are automatically identified as exit points.
+The input model is converted as a whole if neither *`--input`* nor *`--output`* command line options are used. All *`Placeholder`* operations in a TensorFlow graph are automatically identified as entry points. The *`Input`* layer type is generated for each of them. All nodes that have no consumers are automatically identified as exit points.
 
-For Inception_V1, there is one `Placeholder`: input. If the model is viewed in the TensorBoard\*, the input operation is easy to find:
+For Inception_V1, there is one *`Placeholder`*: input. If the model is viewed in the TensorBoard, the input operation is easy to find:
  
 ![Placeholder in Inception V1](../../img/inception_v1_std_input.png)
 
-There is only one output operation, which enclosed in a nested name scope `InceptionV1/Logits/Predictions`, the `Reshape` operation has a full name `InceptionV1/Logits/Predictions/Reshape_1`.
+There is only one output operation, which is enclosed in a nested name scope *`InceptionV1/Logits/Predictions`*. The *`Reshape`* operation has a full name *`InceptionV1/Logits/Predictions/Reshape_1`*.
 
-In the TensorBoard, it looks the following way together with some predecessors:
+In the TensorBoard, along with some of its predecessors, it looks as follows:
 
 ![TensorBoard with predecessors](../../img/inception_v1_std_output.png)
 
 Convert this model and put the results in a writable output directory:
 ```sh
 mo --input_model inception_v1.pb -b 1 --output_dir <OUTPUT_MODEL_DIR>
 ```
-(The other examples on this page assume that you first cd to the `model_optimizer` directory and add the `--output_dir` argument with a directory where you have write permissions.)
+(The other examples on this page assume that you first cd to the *`model_optimizer`* directory and add the *`--output_dir`* argument with a directory where you have read/write permissions.)
```

Comment:
```suggestion
(The other examples on this page assume that you first *`cd`* to the *`model_optimizer`* directory and add the *`--output_dir`* argument with a directory where you have read/write permissions.)
```
Maybe another thing to talk about in whole group, but I think that cmd commands also should be put in *` `*. I think without it they look a little bit out of place.